### PR TITLE
CAPP Agile Learning V2

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Goals:

- more clear what CAPP Agile Learning can do for your organization
- more balance in the features (with flexbox)
- conversion for demo in the top of the page

Which must lead to more conversions at a higher conversion rate. 

### Todo:

- [x] New hero with new image and conversion form
- [x] Split partials
- [x] New Dutch copy
- [x] New Dutch screenshots 
- [ ] New German copy
- [ ] New German screenshots
- [ ] New English copy
- [ ] New English screenshots
- [x] Icons in the why partial
- [ ] Remove unused images

### Screenshots: 

Compare with https://www.defacto.nl/capp-agile-learning

capp agile learning - defacto

### CODE REVIEW

File: source/localizable/_hero_capp-agile.erb [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
   <div class="hero-inner">
+
     <div class="hero-copy">
 
-      <%= image_tag "logos/capp-agile-alt.svg", alt: "CAPP Agile Learning logo", class: "logo" %>
+      <%= image_tag "logos/capp-agile-alt.svg", alt: "CAPP Agile Learning logo", class: "agile-logo" %>
+
+      <h1><%= t("capp-agile.hero.heading") %> <span><%= t("capp-agile.hero.14_days") %></span></h1>
+
+      <!-- <a class="button outline" href="#"><%= t("capp-agile.hero.watch") %></a> -->
```
Comment (fatboypunk) [inline source/localizable/_hero_capp-agile.erb]: No need for a comment

**Comment by RonaldDijkstra** [comment]
Do not forget to update the German and English copy, after Dutch is good enough.

**Comment by sn3p** [comment]
Wanne finish this together, maybe in a Friday paring session 👯‍♂️

**Comment by RonaldDijkstra** [comment]
We need better content first.

**Comment by RonaldDijkstra** [comment]
Abandoned.

**Pull request closed without merging**